app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is List of Post for all users</h1>
\ No newline at end of file
+<section class="container">
+    <%= render partial: 'shared/user', locals: { user: @user } %>
+    <% @posts.each do |post| %>  
+        <div class="row mt-4">
+            <div class="col-lg-12 border border-dark">
+                <h4 class="pt-2"><%= link_to post.title, user_post_path(@user, post), class: "text-dark" %></h4>
+                <p class="mt-2"><%= truncate(post.text, length: 200, omission: '...') %></p>
+                <p class="d-flex justify-content-end">Comments <%= post.comments_counter %>, Likes <%= post.likes_counter%></p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-lg-12 border border-dark">
+                <% post.comments.each do |comment| %>
+                    <p class=""><%= comment.author.name %>: <%= comment.body %> </p>
+                <% end %>
+
+            </div>
+        </div>
```

Comment:
- [ ] Please use the method `recent_comments`  available in the `post` model and loop through it instead of looping through all comments.